.github/workflows/release-github.yml

```diff
@@ -84,3 +84,13 @@ jobs:
           generate_release_notes: true
           body: ${{ steps.release_notes.outputs.body }}
           token: ${{ secrets.GITHUB_TOKEN }}
+  scan-secrets:
+    needs: release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v5
+      - name: Secrets scanning
+        run: |
+          curl -X POST https://api.github.com/repos/digicatapult/shared-workflows/dispatches \
+            -d '{"event_type": "secrets"}' \
+            -H "Authorization: token $GITHUB_TOKEN"
```

Comment:
This is very unlikely to work.

The shared workflow will execute in its own repository, instead of in the calling repository if we do this.